packages/blocks/src/blocks.ts

```diff
@@ -68,22 +83,65 @@ export const Divider = (
     limiterOverrides,
   );
 
-// --- File --- https://api.slack.com/reference/block-kit/blocks#file
-
-// {
-//   block_id: [255, disallow],
-// }
-
-// --- Image --- https://api.slack.com/reference/block-kit/blocks#image
+/**
+ * Displays a [remote file](https://api.slack.com/messaging/files/remote).
+ *
+ * Note: only supported in Messages surface (not in modals or home tabs)
+ *
+ * https://api.slack.com/reference/block-kit/blocks#file
+ */
+export const FileRemote = (
+  external_id: string,
+  block_id?: string,
+  limiterOverrides?: LimiterFuncs,
+): FileBlock =>
+  applyLimitersWithOverrides<FileBlock>(
+    {
+      type: 'file',
+      source: 'remote',
+      external_id,
+      block_id,
+    },
+    { block_id: [255, disallow] },
+    limiterOverrides,
+  );
 
-// {
-//   image_url: [3000, truncate],
-//   alt_text: [2000, ellipsis],
-//   title: [2000, ellipsis],
-//   block_id: [255, disallow],
-// }
+/**
+ * Image: A simple image block, designed to make those cat photos really pop.
+ *
+ * https://api.slack.com/reference/block-kit/blocks#image
+ */
+export const ImageBl = (
```

Comment:
So, for naming, I think this is `Image`, `ImageEl` should be `ImageElement`. 

CA: 4 I don't like `Bl` and `El`.

I think giving the `block` the "real name" makes sense, we also have precedent for adding the whole `Element` on to other things

Reply:
`ImageElement` clashes with Slack's typings. So does `ImageBlock`. 😞 

> I don't like Bl and El.

As I mentioned in our talk earlier, this was also my problem with this change, don't have a great idea otherwise though. Could be convinced of `Image` for block and `ImageEl` for the element? That's still not ideal. I am documenting them better as we speak, but still. Looking into it more now.

Reply:
After discussing, going to make this not a breaking change and have the `ImageBlock` match the Slack typing. 

Can update the interface in the next major release, since this is getting reworked currently.